### DISCUSSION TITLE
The changelog file cannot be viewed on the web page.

### DISCUSSION THREAD

**Discussion by ryo3552130** [community/community#28412] [Other Feature Feedback, Questions, & Ideas] [answered]
In below repository, I cannot view [CHANGELOG.rst](https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst) file becase of "This page is taking too long to load."error.

https://github.com/aws/aws-cli/tree/v2

Are there any problems or specification changes on this web page?
Thank you.

**Comment by radar-blip** [comment]
It opened for me if I right click and "open in a new window".

Otherwise it wouldn't open at all.

**Reply by ryo3552130** [reply]
>It opened for me if I right click and "open in a new window".

I cannnot open even if open it in new windows.
What browser and version do you use?(I try firefox and 91.11.0esr (64-bit) and safari 15.5 on MAC OS, but I cannot open it)

**Reply by radar-blip** [reply]
Im using Firefox 103.0.1  on linux.

**Reply by ryo3552130** [reply]
I try Firefox 103.0.1 on windows, but I cannot view changelog file.
I also checked with my colleague, but he couldn't open it as well.

So, there may be some other factor.(example: geo location.I try it from japan.)

**Reply by radar-blip** [reply]
Now its taking to long to load, wont open at all.

**Reply by ryo3552130** [reply]
+1

**Comment by ryo3552130** [comment]
And I can view develop branch's changelog file. Only v2 branch's changelog file cannot be viewed.

https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst

**Comment by 3kh0** [comment, marked as answer]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Comment by jfullstackdev** [comment]
that's quite a big file,

![image](https://user-images.githubusercontent.com/47092464/194710292-e843f919-f652-4146-8560-091931892ebf.png)

it will depend on your Internet connection, memory, etc. 

I'm using fiber, that took a while but successfully loaded
